protocols/66e60f/66e60f.py

```diff
@@ -0,0 +1,301 @@
+    "source_type":"biorad_96_wellplate_200ul_pcr",
+    "dest_type":"biorad_96_wellplate_200ul_pcr"}""")
+    return [_all_values[n] for n in names]
+
+
+def run(ctx: protocol_api.ProtocolContext):
+
+    [input_csv, source_type, dest_type] = get_values(  # noqa: F821
+        'input_csv', 'source_type', 'dest_type')
+
+    # define all custom variables above here with descriptions:
+    left_pipette_loadname = 'p20_single_gen2'
+    right_pipette_loadname = 'p300_single_gen2'
+
+    target_plate_loader = (dest_type, '1',
+                           'target plate')
+    DNA_sample_plate_loader = (dest_type, '7',
+                               'DNA sample plate')
+    tiprack_300uL_loader = ('opentrons_96_filtertiprack_200ul', '2')
+    tiprack_20uL_loader = ('opentrons_96_filtertiprack_20ul', '5')
+
+    reservoir_loader = ('nest_12_reservoir_15ml', '4', 'water reservoir')
+
+    # Initial 40 uL water for each well of the target
+    initial_water_volume = 40
+
+    # Read CSV and format the inputs
+    # csv format: Well | Description | Concentration | volume to transfer
+    #              [0]       [1]           [2]                [3]
+    data = [
+        [val.strip().upper() for val in line.split(',')
+            if val != '']
+        for line in input_csv.splitlines()[1:]
+        if line and line.split(',')[0]]
+
+    # Convert any well designation in column 1 from [A-H]0[1-9] to [A-H][1-9]
+    # e.g. A01 -> A1 etc.
+    pattern = re.compile('[A-H]0[1-9]')
```

Comment:
Could be less computationally intensive, e.g. check if the second character is a 0
Nick: return well[0] + str(int(well[1:]))